src/components/swap/stockGraph.vue

```diff
@@ -1,142 +1,234 @@
         color: 'red',
         is3D: true,
-        colors:['#2fd7fe'],
-        vAxis:{
-          gridlines:{
-            count: 0
+        colors: ['#2fd7fe'],
+        vAxis: {
+          gridlines: {
+            count: 0,
           },
-          textStyle:{
-            color: '#e3def7'
+          textStyle: {
+            color: '#e3def7',
           },
-            format: 'currency',
-          
         },
-        hAxis:{
-          gridlines:{
-            count: 0
+        hAxis: {
+          gridlines: {
+            count: 0,
           },
-          textStyle:{
-            color: '#e3def7'
+          textStyle: {
+            color: '#e3def7',
           },
-          viewWindowMode: 'maximized'
+          viewWindowMode: 'maximized',
+          showTextEvery: 4,
+        },
+        legend: {
+          maxLines: 2,
+          position: 'top',
+          textStyle: { color: 'white', fontSize: 16 },
         },
-        legend: 'none',
         explorer: {
           axis: 'horizontal',
           keepInBounds: 'true',
-          maxZoomOut: 1
+          maxZoomOut: 1,
         },
         dataOpacity: 0.3,
         title: 'Comparison',
         titleTextStyle: {
-          color: "white",
+          color: 'white',
+        },
+      },
+    };
+  },
+  computed: {
+    ...mapGetters('swap', [
+      'getFromToken',
+      'getToToken',
+      'getIsValidPair',
+      'getPool',
+    ]),
+    fromTokenSymbol() {
+      return this.getFromToken?.symbol;
+    },
+    toTokenSymbol() {
+      return this.getToToken?.symbol;
+    },
+    fromAndToToken() {
+      return `${this.getFromToken?.symbol}|${this.getToToken?.symbol}`;
+    },
+    graphHeaders() {
+      return ['Time', `${this.fromTokenSymbol}\\${this.toTokenSymbol}`];
+    },
+  },
+  methods: {
+    getDateFormatGraph(date) {
+      switch (this.timeSeries) {
+        case 'daily':
+          return moment.utc(date).format('MMM DD, HH:mm');
+        default:
+          return moment.utc(date).format('MMM DD, YYYY');
+      }
+    },
+    pricePerToken() {
+      const token0 = this.getPool.reserve0;
+      const token1 = this.getPool.reserve1;
+      if (token0.symbol === this.fromTokenSymbol) {
+        return Number(
+          parseFloat(token0.quantity / token1.quantity).toFixed(
+            token0.precision
+          )
+        );
+      }
+      return Number(
+        parseFloat(token1.quantity / token0.quantity).toFixed(token1.precision)
+      );
+    },
+    async getServerTime() {
+      try {
+        const response = await axios.get(
+          `${process.env.BACKEND_ENDPOINT}/?action=getCurrentTime`
+        );
+        const serverTime = response.data && response.data[0];
+        if (serverTime) {
+          debugModeOn && console.log('getServerTime', serverTime.CURRENTTIME);
+          this.currentServerTime = new Date(serverTime.CURRENTTIME);
         }
+      } catch {
+        this.$errorNotification('Error connecting to Database');
```

Comment:
I think we can make this a silent catch without an error notification. Maybe just do a `console.error`

Reply:
Fixed